Vagrantfile

```diff
@@ -24,94 +25,20 @@ Vagrant.configure("2") do |config|
     config.vm.define node[:name] do |cplane|
       cplane.vm.box = node[:box]
       cplane.vm.network node[:network], ip: node[:ip]
-      cplane.vm.network "forwarded_port", guest: 443, host: 8443 # Port Forward for k8s dashboard
       cplane.vm.hostname = node[:name]
       cplane.vm.provider "virtualbox" do |v|
         v.name = node[:name]
         v.memory = 2048
         v.cpus = 2
       end
       cplane.vm.provision "shell",
-        env: {
-          "ETC_HOSTS"     => ETC_HOSTS,
-          "BASE_CIDR"     => BASE_CIDR,
-          "API_SERVER_IP" => node[:ip] # API Server is the control plane host itself
-        },
+        env: { "ETC_HOSTS" => ETC_HOSTS },
         inline: <<-SHELL
```

Comment:
The ENTIRE provisioning inline script was extracted and moved to the script `scripts/provision/provision_base.sh`.  This is the same for both the cplane and worker configs